core/src/raw/tests/utils.rs

```diff
@@ -59,11 +59,14 @@ pub fn init_test_service() -> Result<Option<Operator>> {
     // Use random root unless OPENDAL_DISABLE_RANDOM_ROOT is set to true.
     let disable_random_root = env::var("OPENDAL_DISABLE_RANDOM_ROOT").unwrap_or_default() == "true";
     if !disable_random_root {
-        let root = format!(
-            "{}{}/",
-            cfg.get("root").cloned().unwrap_or_else(|| "/".to_string()),
-            uuid::Uuid::new_v4()
-        );
+        let mut root_path = cfg.get("root").cloned().unwrap_or_else(|| "/".to_string());
+
+        // Ensure root path ends with "/"
```

Comment:
for simplicity, we could use this approach: 
```suggestion
        normalize_path(&cfg.get("root").cloned().unwrap_or_default())
```
I'm not sure if we could simply format the root as `{}/{}/`, since not all services may have called `normalize_path(root)` internally.

Reply:
Done thanks